src/vs/workbench/contrib/welcome/page/browser/welcomePage.contribution.ts

```diff
@@ -23,15 +23,15 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 			'workbench.startupEditor': {
 				'scope': ConfigurationScope.APPLICATION, // Make sure repositories cannot trigger opening a README for tracking.
 				'type': 'string',
-				'enum': ['none', 'welcomePage', 'readme', 'newUntitledFile', 'welcomePageInEmptyWorkbench'],
+				'enum': ['none', 'welcomePage', 'welcomePageWithTour', 'readme', 'newUntitledFile', 'welcomePageInEmptyWorkbench'],
```

Comment:
Add a description for this new value as well

Reply:
I believe this has been fixed